src/auth/ntlm/Scheme.cc

```diff
@@ -9,16 +9,27 @@
 #include "squid.h"
 #include "auth/ntlm/Config.h"
 #include "auth/ntlm/Scheme.h"
+#include "base/RunnersRegistry.h"
 #include "debug/Messages.h"
 #include "debug/Stream.h"
 #include "helper.h"
 
-Auth::Scheme::Pointer Auth::Ntlm::Scheme::_instance = NULL;
+class NtlmAuthRr : public RegisteredRunner
+{
+public:
+    void bootstrapConfig() {
```

Comment:
Please be explicit about this method being a parent's method override.

```suggestion
    /* RegisteredRunner API */
    virtual void bootstrapConfig() override {
```